Makefile

```diff
@@ -59,5 +59,4 @@ check-headers:
 
 .PHONY: check-tidy
 check-tidy: generate
-	go mod tidy
```

Comment:
where did this go? Shouldn't `check-tidy` run `go mod tidy`?

Reply:
This is unfortunately breaking CI right now, see 99designs/gqlgen#1483: The `go generate` steps requires entries in go.sum, and reports errors if they are missing. `go mod tidy` then removes them. So it is currently impossible to commit code that satisfies both

Reply:
😭